tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -332,3 +336,67 @@ def test_positive_validate_without_req_file_output(self):
 
         :CaseAutomation: notautomated
         """
+
+    @tier1
+    def test_positive_validate_capsule_certificate(self):
+        """Check that Capsules cert handles additional proxy names.
+
+        :id: 8b53fc3d-704f-44f4-899e-74654529bfcf
+
+        :steps:
+
+            1. Generate a Capsule certificate
+            2. Confirm proxy server's FQDN for DNS is present
+            3. Confirm that format of alternative names does not include []
+
+        :expectedresults: Capsule certs has valid DNS values
+
+        :BZ: 1747581
+
+        :CaseAutomation: automated
+        """
+        DNS_Check = False
+        tmp_dir = '/var/tmp/{0}'.format(gen_string('alpha', 4))
+        cert_file = '{0}/ssl-build/capsule.example.com/cert-data'.format(tmp_dir)
+        if not os.path.exists('{0}/ssl-build/capsule.example.com/'.format(tmp_dir)):
+            os.makedirs('{0}/ssl-build/capsule.example.com/'.format(tmp_dir))
```

Comment:
Just call `os.makedirs('...'.format(tmp_dir), exist_ok=True)`.

Reply:
or if we switch to using Pathlib..
```python
Path(temp_dir).mkdir(parents=True, exist_ok=True)
```